dd-trace-ot/src/main/java/datadog/opentracing/DDSpanContext.java

```diff
@@ -45,6 +47,9 @@
   /** Tags are associated to the current span, they will not propagate to the children span */
   private final Map<String, Object> tags = new ConcurrentHashMap<>();
 
+  /** Logs are associated to the current span, they will not propagate to the children span */
+  private final Map<Long, List<Map<String, ?>>> logs = new ConcurrentHashMap<>();
```

Comment:
Does this really need to be a map indexed by timestamp? It could simply be a `List<Pair<Long, Map<String, ?>>>`

Reply:
I'd prefer not to introduce a new dependency (the project isn't currently using apache commons), so used `SimpleEntry` instead.  The list doesn't need to be instantiated via `synchronizedList()`, right?

Reply:
If DDSpanContext isn't expected to be thread-safe, then no.